src/index.ts

```diff
@@ -98,17 +127,15 @@ const converterRequiredFields = {
     toZigbee: 'Array',
 };
 
-function validateDefinition(definition: Definition) {
+function validateDefinition(definition: Definition): asserts definition is Definition {
```

Comment:
I think the validateDefinition can be made optional, only has to be done for external converters. Official converters are already covered by typescript, this stems from the JS era 😄  Should give some more perf improvements!